clients/python/README.md

```diff
@@ -0,0 +1,164 @@
+# samara-sdk
+
+A python client sdk for Samsara.
+
+## Usage
+
+To use the Samsara SDK, pip install it via:
+
+```bash
+$ pip install samsara-sdk
+```
+
+To bring it into the repl:
+
+```python
+> from samsara-sdk import SamsaraClient
+```
+
+And then to configure:
+
+```python
+> client = SamsaraClient(url="https://my.samsara.server:9000/", source_id="source identifier")
+```
+
+
+Now you can publish events to samsara!
+
+### Publish an Event
+
+Samsara SDK buffers events and published them to the Ingestion API
+periodically. To record an event in the Samasara buffer:
+
+
+```python
+> client.record_event(event_name="user.logged", source_id="device1", timestamp=1234567890)
```

Comment:
here you should make clear that `timestamp` is in **milliseconds** and not *seconds* format.
In other words putting UNIXTIME won't work. maybe worthy to show how to get the milliseconds time here
as this is a common source of confusion.